.travis.yml

```diff
@@ -25,11 +25,11 @@ jobs:
     script: make test
 
   - stage: Build docker
-    deploy: 
+    deploy:
       on:
         tags: false
       provider: script
-      script: VERSION=${TRAVIS_COMMIT} make build-docker
+      script: REVISION=${TRAVIS_COMMIT} make build-linux; VERSION=${TRAVIS_COMMIT} make build-docker
```

Comment:
You could run multiple `make` targets if you want to, e.g. `make build-linux build-docker` but if you stick to this maybe use `&&` instead of `;` cause the latter should take place only if the first succeeds.